week-4/C-more-complex-objects/exercise-1.js

```diff
@@ -10,8 +10,22 @@ var house = {
     currentOwner: {
       firstName: "Margaret",
       lastName: "Conway"
+    },
+    newAddress: function(newAddress){
+      this.address = newAddress;
+    },
+    NewPreviousOwners: function(newOwner){

```

Comment:
I got quite confused by the argument name `newOwner`. Can you think of a better name?

Reply:
@foaly-nr1 , Sure will do